components/CardList/index.tsx

```diff
@@ -29,23 +31,85 @@ const CardListPublisherCard = ({ items }: { items: CardProps[] }) => {
   );
 };
 
-const CardListTopicCard = ({ items }: { items: CardProps[] }) => {
+const CardListSubtopicCard = ({
+  items,
+  parentTopic,
+}: {
+  items: { title: string; description: string; "@id": string }[];
+  parentTopic: { title: string; "@id": string };
+}) => {
+  const getDatasetHref = (id: string) => {
+    return (
+      "/datasets?topic=" +
+      encodeURIComponent(parentTopic.title) +
+      "&subtopics=" +
+      encodeURIComponent(id)
+    );
+  };
+
   return (
     <ul className="app-cards__list" data-track-count="cardList">
-      {items.map((item: CardProps) => (
-        <li className="app-cards__list-item--topic" key={item.heading}>
-          <div className="app-cards__list-item-wrapper--topic">
-            <h2 className="app-cards__sub-heading govuk-heading-s">
-              <a className="govuk-link app-cards__link--topic" href={item.href}>
-                {item.heading}
-              </a>
-            </h2>
-            <p className="govuk-body app-cards__description">
-              {item.description}
-            </p>
-          </div>
-        </li>
-      ))}
+      {items.map(
+        (
+          item: { title: string; description: string; "@id": string },
+          index: number
+        ) => (
+          <li className="app-cards__list-item--topic" key={item.title + index}>
+            <div className="app-cards__list-item-wrapper--topic">
+              <h2 className="app-cards__sub-heading govuk-heading-s">
+                <a
+                  className="govuk-link app-cards__link--topic"
+                  href={getDatasetHref(item.title)}
+                >
+                  {item.title}
+                </a>
+              </h2>
+              <p className="govuk-body app-cards__description">
+                {item.description}
+              </p>
+            </div>
+          </li>
+        )
+      )}
+    </ul>
+  );
+};
+
+const CardListTopicCard = ({
+  items,
+}: {
+  items: { title: string; description: string; "@id": string }[];
+}) => {
+  const getTopicHref = (id: string) => {
+    const splitTopic = id.split("/");
```

Comment:
if I'm reading this right you can get what you want from the "identifier" field, have corrected the data now (in pr as a write this) so it should work (and will always be the case - we just need more data validation).